core/gui/src/app/dashboard/user/component/share-access/share-access.component.html

```diff
@@ -68,10 +68,10 @@
 <ul
   class="current-share"
   id="current-share">
-  <li>{{ owner }} <span class="badge badge-primary">Owner</span></li>
+  <li>{{ owner }} <nz-tag nzColor="green">OWNER</nz-tag></li>
   <li *ngFor="let entry of accessList">
     {{ entry.email }} ({{ entry.name }})
-    <span class="badge badge-primary">{{ entry.privilege }}</span>
+    <nz-tag nzColor="blue">{{ entry.privilege }}</nz-tag>
```

Comment:
move the tag before the user account email?

Reply:
Good idea!